dask/dataframe/core.py

```diff
@@ -4827,6 +4838,46 @@ def query(self, expr, **kwargs):
         See also
         --------
         pandas.DataFrame.query
+        pandas.eval
+
+        Examples
+        --------
+        >>> import pandas as pd
+        >>> import dask.dataframe as dd
+        >>> df = pd.DataFrame({'x': [1, 2, 1, 2],
+        ...                    'y': [1, 2, 3, 4],
+        ...                    'z z': [4, 3, 2, 1]})
+        >>> ddf = dd.from_pandas(df, npartitions=2)
+
+        Refer to column names directly:
+
+        >>> ddf.query('y > x').compute()
+           x  y  z z
+        2  1  3    2
+        3  2  4    1
+
+        Refer to column name using `backticks`:
+
+        >>> ddf.query('`z z` > x').compute()
+           x  y  z z
+        0  1  1    4
+        1  2  2    3
+        2  1  3    2
+
+        Refer to variable name using f-strigs:
+
+        >>> value = 1
+        >>> ddf.query(f'x == {value}').compute()
+            x  y  z z
+        0  1  1    4
+        2  1  3    2
```

Comment:
The `x y z z` header needs to be moved over to the left a bit so it aligns with the data below (similar to the two `query` examples above).